lib/MusicBrainz/Server/Data/Collection.pm

```diff
@@ -274,10 +274,13 @@ sub find_by {
         push @args, $editor_id, $editor_id;
     } elsif ($editor_id = $opts->{show_private_only}) {
```

Comment:
OK, I hadn't looked at this code in a while and was confused what this was for (was wondering why we are returning all private collections other than an editor's own to that editor) but it's just to get the "hidden" count. In the future, I feel it'd be safer to have a separate method that just returns the count, since the actual collections aren't used AFAICT.

Reply:
You do have a very good point - pushed a second commit here that should do the trick.

Reply:
didn't test the new method but lgtm :shipit: